config/routes.rb

```diff
@@ -86,7 +86,8 @@
 
     get '/apply' => 'claims#new'
     get '/apply/refund' => 'refunds#new'
-    get '/apply/users', to: redirect('/apply/application-number')
```

Comment:
I would have thought that you can just as easily get /apply/users - why not leave this in ?